custom_components/tuya_local/devices/momcozy_white_noise_machine_v2.yaml

```diff
@@ -0,0 +1,300 @@
+        name: volume_level
+        type: integer
+        range:
+          min: 10
+          max: 100
+        mapping:
+          - scale: 100
+      - id: 10
+        name: tone
+        type: string
+        mapping:
+          - dps_val: "1"
+            value: White noise
+          - dps_val: "2"
+            value: Pink noise
+          - dps_val: "3"
+            value: Brown noise
+          - dps_val: "4"
+            value: Wall fan
+          - dps_val: "5"
+            value: Fan
+          - dps_val: "6"
+            value: Floor fan
+          - dps_val: "7"
+            value: Wind
+          - dps_val: "8"
+            value: Rain on roof
+          - dps_val: "9"
+            value: Rain
+          - dps_val: "10"
+            value: Thunderstorm
+          - dps_val: "11"
+            value: Light rain
+          - dps_val: "12"
+            value: Ocean wave
+          - dps_val: "13"
+            value: Water stream
+          - dps_val: "14"
+            value: Birds
+          - dps_val: "15"
+            value: Crickets
+          - dps_val: "16"
+            value: Frog
+          - dps_val: "17"
+            value: Fireplace
+          - dps_val: "18"
+            value: Campfire
+          - dps_val: "19"
+            value: Womb
+          - dps_val: "20"
+            value: Shushing
+          - dps_val: "21"
+            value: Cafe
+          - dps_val: "22"
+            value: Maracas
+          - dps_val: "23"
+            value: Clock
+          - dps_val: "24"
+            value: Buddha
+          - dps_val: "25"
+            value: Wind chimes
+          - dps_val: "26"
+            value: Steamship
+          - dps_val: "27"
+            value: Aircraft
+          - dps_val: "28"
+            value: Train
+          - dps_val: "29"
+            value: Lullaby
+          - dps_val: "30"
+            value: Music box
+          - dps_val: "31"
+            value: Twinkle little star
+          - dps_val: "32"
+            value: Good morning
+          - dps_val: "33"
+            value: Fantasy
+          - dps_val: "34"
+            value: Joyful
+      - id: 20
+        name: duration
+        type: integer
+        unit: min
+        range:
+          min: 0
+          max: 1440
+  - entity: select
+    name: Favorites
+    icon: "mdi:palette"
+    category: config
+    dps:
+      - id: 13
+        type: string
+        name: option
+        optional: true
+        mapping:
+          - dps_val: mode_1
+            value: Scene 1
+          - dps_val: mode_2
+            value: Scene 2
+          - dps_val: mode_3
+            value: Scene 3
+          - dps_val: mode_4
+            value: Scene 4
+          - dps_val: mode_5
+            value: Scene 5
+          - dps_val: mode_6
+            value: Scene 6
+          - dps_val: mode_7
+            value: Scene 7
+          - dps_val: mode_8
+            value: Scene 8
+          - dps_val: null
+            value: None
+  - entity: number
```

Comment:
As with the v1 config, volume, tone and duration do not need to be duplicated here.